src/init/autoDetectProject.ts

```diff
@@ -66,7 +61,9 @@ export function iconsDisabled(name: string): boolean {
 export function isProject(projectJson: any, name: string): boolean {
   switch (name) {
     case 'ng':
-      return (projectJson.dependencies && (projectJson.dependencies['@angular/core'] != null)) || false;
+      return (projectJson.dependencies && (projectJson.dependencies['@angular/core'] != null)) ||
+        (projectJson.devDependencies && (projectJson.devDependencies['@angular/core'] != null)) ||
+        false;
```

Comment:
Consider it a force of habit to add `guard clause` for `null`. In this case, it's indeed redundant as we already check for `null`. Nice catch.